site/docs/reference/Connectors/capture-connectors/twilio.md

```diff
@@ -1,3 +1,8 @@
+
```

Comment:
Whenever you add sidebar_position, make sure you start on line 1 of the file. Otherwise, docusaurus doesn't see it, so it doesn't work AND it prevents the title from rendering properly in the TOC (you can also tell by the grey text color when it's not working).

Reply:
Thanks for explaining how the `sidebar_position` works! During my last commit, there were a few conflicts suggesting the sidebar_position changes, I think it might be because I used a previously used codespace. I have rolled back the changes to sidebar_position now, those weren't intentional, I'll let you handle this. Also, will get with you to see if I can also render previews on my side, thanks!